scope & closures/ch5.md

```diff
@@ -81,7 +81,7 @@ Após a execução de `foo()`, normalmente nós esperamos que o conjunto do esco
 
 Mas a "mágica" das closures não permite que isso aconteça. Esse escopo interno, de fato, *ainda* está "em uso" e, portanto, ele não desaparece. Quem está usando? **A função `baz()`**.
 
-By virtue of where it was declared, `bar()` has a lexical scope closure over that inner scope of `foo()`, which keeps that scope alive for `bar()` to reference at any later time.
+Em virtude de onde foi declarada, `bar()` tem uma closure sobre o escopo léxico do escopo interno de `foo()`, que mantem o escopo vivo para `bar()` fazer referência a qualquer momento posterior.
```

Comment:
1 - Fiquei com dúvida na parte `'bar()' has a lexical scope closure over that inner scope of 'foo()'`. Principalmente em **lexical scope closure**. Pensei em duas possibilidades além da sua, mas não tenho certeza se estou ajudando ou atrapalhando.
* `tem um escopo lexical closure sobre esse escopo interno de 'foo()'`; ou
* `tem uma closure sobre esse escopo interno de 'foo()'`.

2 - Trocar **o** entre mantem e escopo por **esse**. `que mantem esse escopo vivo(...)`.